app/controllers/multifactor_auths_controller.rb

```diff
@@ -37,6 +43,50 @@ def update
     end
   end
 
+  def update
+    unless valid_mfa_level?
+      redirect_to edit_settings_path, flash: { error: t(".invalid_level") }
+      return
+    end
```

Comment:
TODO: this can be moved to `handle_new_level_param` as that method checks the `mfa_level` value